lib/gdal/color_table.rb

```diff
@@ -138,6 +135,7 @@ def color_entry_as_rgb(index)
     # @param four [Integer] The `c4` value of the GDAL::ColorEntry
     #   struct to set.
     # @return [GDAL::ColorEntry]
+    # rubocop:disable Metrics/ParameterLists
```

Comment:
As long as we're making breaking changes, how about switching to named params?

Reply:
I don't think I care on this one, but I thought about that for this method and it seemed a bit more natural to me to call it like `thing.add_color_entry(1, 10, 20, 30)` as opposed to `thing.add_color_entry(1, one: 10, two: 20, three: 30)`. But I don't have strong feelings so I'll change.

Reply:
Actually, I think it might make sense to scan the whole library for these and change them all at once.

Reply:
I think for this method in particular named params are more natural since it has the logic where it only sets the colors if they exist.  So you could `thing.add_color_entry four: 40` instead of `thing.add_color_entry nil, nil, nil, 40`